### PR TITLE
feat(vsc): new create project UI

### PR DESCRIPTION
**Pull request by tecton**
Feature: https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/18028376

Question layout only changes in VS Code extension. Question option strings are also updated for CLI and VS.

Effect:
image

### CODE REVIEW

File: packages/fx-core/resource/package.nls.json [hunk 1]
```diff
@@ -461,46 +461,59 @@
...
   "core.ExistingTabEndpointQuestion.placeholder": "Input your existing tab endpoint",
   "core.ExistingTabEndpointQuestion.title": "Enter your web app's local endpoint",
   "core.ExistingTabEndpointQuestion.validation": "Please enter a valid URL started with https://",
   "core.TabSPFxOption.label": "Tab (SPFx)",
-  "core.TabSPFxOption.labelNew": "SPFx tab",
+  "core.TabSPFxOption.labelNew": "SPFx",
   "core.TabSPFxOption.description": "Client-side app with SharePoint Framework",
   "core.TabSPFxOption.detail": "Auto-hosted SPFx web part tightly integrated with Microsoft Teams",
-  "core.TabSPFxOption.detailNew": "SPFx components hosted in Microsoft 365 and embeded in Microsoft Teams",
+  "core.TabSPFxOption.detailNew": "Build UI with Sharepoint Framework",
```
Comment (HuihuiWu-Microsoft) [inline packages/fx-core/resource/package.nls.json:491]: Sharepoint -> SharePoint

Comment (tecton) [reply packages/fx-core/resource/package.nls.json]: Thanks! updated.

**Review by HuihuiWu-Microsoft — DISMISSED** [review]

File: packages/fx-core/resource/package.nls.json [hunk 2]
```diff
@@ -590,10 +603,10 @@
   "core.aadAppQuestion.description": "Your Azure Active Directory app for Single Sign On",
   "core.teamsAppQuestion.label": "Teams app",
   "core.teamsAppQuestion.description": "Your Teams app", 
-  "core.M365SsoLaunchPageOptionItem.label": "SSO enabled personal tab",
-  "core.M365SsoLaunchPageOptionItem.detail": "Identity aware webpages embedded in Microsoft Teams, Outlook and the Microsoft 365 app",
-  "core.M365SearchAppOptionItem.label": "Search-based message extension",
-  "core.M365SearchAppOptionItem.detail": "Insert search result when user composes messages in Microsoft Teams or emails in Outlook",
+  "core.M365SsoLaunchPageOptionItem.label": "React with Fluent UI",
+  "core.M365SsoLaunchPageOptionItem.detail": "A web app that uses Fluent UI React components to get a Teams look and feel",
+  "core.M365SearchAppOptionItem.label": "Collect Form Input and Process Data",
```
Comment (sffamily) [inline packages/fx-core/resource/package.nls.json:608]: Other labels only capitalize the first character, need to be consistent. This label is in CamelCase,

Comment (tecton) [reply packages/fx-core/resource/package.nls.json]: @sffamily Thanks, I missed this one and fixed now.

**Review by sffamily — APPROVED** [review]

**Review by HuihuiWu-Microsoft — APPROVED** [review]

**Review by 1yefuwang1 — APPROVED** [review]

**Review by jayzhang — APPROVED** [review]

**Pull request merged**